pkg/controller/master/backup/util.go

```diff
@@ -165,3 +169,7 @@ func sanitizeVirtualMachineForRestore(restore *harvesterv1.VirtualMachineRestore
 func getCloudInitSecretRefVolumeName(vmName string, secretName string) string {
 	return fmt.Sprintf("vm-%s-%s-volumeref", vmName, secretName)
 }
+
+func getVMBackupMetadataFileName(vmBackupName string) string {
+	return fmt.Sprintf("%s.cfg", vmBackupName)
```

Comment:
We should consider namespace as well, so `<namespace>-<name>` might be better?

Reply:
Updated it. Thank you.